README.md

```diff
@@ -76,7 +73,7 @@ JSDoc comments had some existing precedence in the JavaScript community for docu
 
 This comment convention is often found in build scripts, small web apps, server-side apps, and elsewhere where
 the cost/benefit tradeoff of adding a build-tool is too high.
-Even when TypeScript doesn't provide type-checking diagnostics, the comment convention is still leveraged in editor functionality because TypeScript powers the underlying JavaScript editing experience.
+Even when no type-checking diagnostics put in place, the comment convention is still leveraged in editors supporting types in their underlying JavaScript editing experience.
```

Comment:
Even when no type-checking diagnostics _are_ put in place
or
Even _with_ no type-checking diagnostics put in place

Reply:
Nevermind, reading this again later with fresh eyes, the original wording seems fine.